style.css

```diff
@@ -1,7 +1,9 @@
 * {
     box-sizing: border-box;
-    user-select: none;
     -webkit-user-drag: none;
+    user-select: none;
+    -moz-user-select: none;
+    -webkit-user-select: none;
```

Comment:
One of those iffy properties that isn't common, so need to cater for other browsers/devices

If you're unsure in the future, bookmark this and type in the property: https://caniuse.com/